### PR TITLE
Run the CI workflow on PRs targeting prep

### PR DESCRIPTION
**Pull request by abcdefg30**
See https://docs.github.com/en/actions/using-workflows/triggering-a-workflow.

### CODE REVIEW

**Review by pchote — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Continuous Integration
 on:
   push:
   pull_request:
-    branches: [ bleed ]
+    branches: [ bleed, 'prep-*' ]
```
Comment (PunkPun) [inline .github/workflows/ci.yml:6]: Why not
```suggestion
    branches: [ bleed, 'prep-**' ]
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Comment (abcdefg30) [reply .github/workflows/ci.yml:6]: Why do we need to match the `/` character?

Comment (PunkPun) [reply .github/workflows/ci.yml:6]: I don't see why we need to exclude it

Comment (abcdefg30) [reply .github/workflows/ci.yml:6]: Prep branches don't or shouldn't contain that character, so I don't see the benefit of using the less obvious operator.

Comment (PunkPun) [reply .github/workflows/ci.yml:6]: If it comes back to bite us in the arse I'll blame you \:P

**Review by PunkPun — APPROVED** [review]

**Pull request merged**